src/thor/astar_bss.cc

```diff
@@ -98,7 +98,7 @@ void AStarBSSAlgorithm::Init(const midgard::PointLL& origll, const midgard::Poin
 // edges if from_transition is false.
 void AStarBSSAlgorithm::ExpandForward(GraphReader& graphreader,
                                       const GraphId& node,
-                                      const EdgeLabel& pred,
+                                      const BDEdgeLabel& pred,
```

Comment:
not sure how to name this better.. this is a unidir algo. it's also not only for a*, we use it in bidir (dijkstra) matrix too. nothing to take action, just wondering out loud.

Reply:
we could name it `astar_label` as its the one that is "directed" ie with the heuristic, to differentiate it from the undirected one. though we do have other labels like mmlabel that are even more algorithm specific so mmlabel is better in that case imho. naming is hard :smile:

Reply:
> we could name it astar_label as its the one that is "directed" ie with the heuristic

like I said, it's also used in costmatrix. it's actually both a bidir edge label and a* edge label so it covers most algos.

anyways, dave actually left a comment. 
```
 * Derived EdgeLabel class used for A* path algorithms and CostMatrix.
 * NOTE - despite the name, this is also used by unidirectional A* algorithms
 * (not simply bidirectional A*).
 ```